phpstan.dist.neon

```diff
@@ -13,4 +13,24 @@ parameters:
         -
             identifier: constructor.unusedParameter
             paths:
+                - src/Behat/Behat/Definition/Printer/ConsoleDefinitionPrinter.php
+                - src/Behat/Behat/EventDispatcher/Cli/StopOnFailureController.php
+                - src/Behat/Behat/Output/Node/Printer/ListPrinter.php
+                - src/Behat/Behat/Tester/Runtime/RuntimeFeatureTester.php
                 - src/Behat/Behat/Transformation/Transformation/ReturnTypeTransformation.php
+                - src/Behat/Testwork/Ordering/Cli/OrderController.php
+        -
+            identifier: function.alreadyNarrowedType
```

Comment:
This error and the next one happen in the event dispatcher that needs to work with both versions of the event dispatcher which change the order of the parameters in the dispatch function, so we need to ignore them